reference/6/Microsoft.PowerShell.Core/About/about_Modules.md

```diff
@@ -334,7 +334,7 @@ To view the default module locations, type:
 $Env:PSModulePath
 ```
 
-To add a default module location, use the following command format.
+To add a default module location on Windows, use the following command format:
 
 ```powershell
 $Env:PSModulePath = $Env:PSModulePath + ";<path>"
```

Comment:
All these examples should use  `+=`  instead; that is, `$Env:PSModulePath += ";<path>"` on Windows, and `$Env:PSModulePath += ":<path>"` on Unix.